### ISSUE TITLE
Feature Request: Show the PIC1.PNG in the shaders compilation screen instead of black screen

### ISSUE THREAD

**Issue by vsub** [RPCS3/rpcs3#4940]
Very low priority.

I just made an autohotkey script that shows the PIC1.PNG in an always on top full screen window before starting the game(the image is visible for 4 seconds which is enough for the shaders to be loaded and then it fades out while you see how the loading is almost done)so I thought it would be nice to see the game image rather than black screen with two loading bars while waiting

Something like this
![2018-07-28_114448](https://user-images.githubusercontent.com/5032540/43355914-22301d06-925e-11e8-9e8e-723c27a3d947.jpg)

**Comment by Megamouse** [comment]
That's a great idea. I also had something like that in mind.
Maybe even with a customization possibility in the future.

**Comment by Hasster1** [comment]
And maybe do something with those "Loading" words, because they can sometimes be the same color, as the picture?

**Comment by Megamouse** [comment]
Well that would be a part of it of course 💡
